framework/src/main/java/org/apache/drill/test/framework/ColumnList.java

```diff
@@ -101,14 +128,17 @@ public String toString() {
       int type = (Integer) (types.get(i));
       if (Simba && (type == Types.VARCHAR)) {
         String s1 = String.valueOf(values.get(i));
-        // if the field has a JSON string or list, then remove newlines so that
-        // each record fits on a single line in the actual output file.
-        // sometimes Drill returns records in different orders, and by
-        // ensuring each reoord is on a single line, they can be matched in any
-        // order
+        // Sometimes Simba returns records that span multiple lines.
+        // If the field has a JSON string or list, then remove newlines
+        // so that each record fits on a single line in the actual output
+        // file.  json/json_kvgenflatten/kvgen/kvgen7.q
+        // Also, sometimes Simba returns varchars with white spaces
+        // in different places.  Remove the white space.
+        // convert/convert32.q
         if ((s1.length() > 0) &&
            ((s1.charAt(0) == '{') || (s1.charAt(0) == '[')) ) {
           s1 = Utils.removeNewLines(s1);
+          s1 = removeStringSpace(s1);
```

Comment:
There might be libraries that do the reverse of _prettifying_ JSON string, including stripping down extra spaces. Consider having a single Util method that does something like `s1 = Utils.compactJson(s1);`